crates/api_models/src/payments.rs

```diff
@@ -3925,6 +3925,9 @@ pub struct PaymentsResponse {
     pub merchant_order_reference_id: Option<String>,
     /// order tax amount calculated by tax connectors
     pub order_tax_amount: Option<MinorUnit>,
+
+    /// Connector Identifier
+    pub connector_mandate_id: Option<String>,
```

Comment:
Should this be "connector mandate identifier" instead?

Reply:
Id stands for identifier right?

Reply:
I should have been clearer, I was referring to the documentation comment.